backend/src/connectors/drive.py

```
@@ -374,25 +374,31 @@ def list_files(self):
+                            owner.get("displayName")
+                            for owner in f.get("owners", [])
+                            if owner.get("displayName")
+                        ],
+                        "mimeType": f["mimeType"],
+                        "modifiedTime": f["modifiedTime"],
+                        "webViewLink": f.get("webViewLink"),
+                        "permissions": self.get_file_principals(f["id"]),
+                    }
+                )
+
+            except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)